source/features/rgh-linkify-yolo-issues.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '../feature-manager';
+import {getRghIssueUrl} from '../helpers/rgh-issue-link';
+import {isRefinedGitHubYoloRepo} from '../github-helpers';
+
+function init(): void {
+	// .js-csv-data is the old selector
+	for (const issueCell of select.all(':is(.js-csv-data, .react-csv-row) td:nth-child(3)')) {
```

Comment:
I think the table is generated on the fly now. 🥲 It only works some of the time


https://user-images.githubusercontent.com/1402241/216777701-a28aa3c8-4f92-42af-8ad7-ded604721c9c.mov




You can replace this select with `observe`. You can look for it and its usage in other files. 


```js
function linkify(issueCell) {
		wrap(issueCell.firstChild!, <a href={getRghIssueUrl(issueCell.textContent!)}/>);
}
function init(signal) {
	observe(':is(.js-csv-data, .react-csv-row) td:nth-child(3)', linkify, {signal});
}
```

Reply:
okay i will give it a try.